@here/harp-examples/src/camera-animations_key-track.ts

```diff
@@ -0,0 +1,220 @@
+/*
+ * Copyright (C) 2017-2020 HERE Europe B.V.
+ * Licensed under Apache 2.0, see full license in LICENSE
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import { GeoCoordinates, mercatorProjection, sphereProjection } from "@here/harp-geoutils";
+import {
+    CameraAnimationBuilder,
+    CameraKeyTrackAnimation,
+    ControlPoint
+} from "@here/harp-map-controls";
+import { CopyrightElementHandler, MapView } from "@here/harp-mapview";
+import { APIFormat, AuthenticationMethod, OmvDataSource } from "@here/harp-omv-datasource";
+import { GUI } from "dat.gui";
+
+import { apikey, copyrightInfo } from "../config";
+
+interface IData {
+    [key: string]: any;
```

Comment:
You can be a slight bit more type safe
```suggestion
    [key: string]: GeoCoordinates;
```

Reply:
done